ui/js/components/perf/compare.js

```diff
@@ -69,6 +69,75 @@ treeherder.component('phCompareTable', {
     }
 });
 
+treeherder.component('phTrendTable', {
+    templateUrl: 'partials/perf/trendtable.html',
```

Comment:
It looks like this is more or less the same as the previous component except for templateUrl. It looks like you can make templateUrl "injectable" by making it a function as opposed to a string: http://stackoverflow.com/a/33846452 So maybe we could just parametrize this part of phCompareTable above and save the duplication? Maybe something like this:

```js
templateUrl: function($element, $attrs) {
   if ($attrs.type === 'trend')
      return 'partials/perf/trendtable.html';
   return 'partials/perf/comparetable.html'
},
```